.changelog/578.bugfix.md

```diff
@@ -0,0 +1 @@
+Fix offline indicators showing ParaTimes are out-of-date after 2 minutes of opening a page
```

Comment:
Linter says this is waaaay to long

Reply:
okay, i am forced to make changelog=commit :shrug: